README.md

```diff
@@ -1,2 +1,5 @@
 # JavascriptDonuts
 Powdered Sugar
+
+add #1
```

Comment:
Line 4 link in [PR](https://github.com/Whats4Dinner/JavascriptDonuts/commit/8224d4bc86aa5f6d5f1246592d815e72d0d8906a)